common/web/types/test/util/test-unescape.ts

```diff
@@ -56,3 +56,9 @@ describe('test unescapeString()', function() {
     assert.throws(() => unescapeString('\\u{110000}'), UnescapeError);
   });
 });
+
+describe('test unescapeOneQuadString()', () => {
+  it('should be able to convert', () => {
+    assert.equal(unescapeOneQuadString('\\u0127'), '\u{0127}');
```

Comment:
Might be a little clearer on the surface if the second string were simply `'ħ'`.

I was fairly confused for a bit until I realized that the critical detail here is that the former case has the escaping-character escaped, but not the latter.

Reply:
Alternative, just add a comment:

```suggestion
    // testing that `\u0127` is unescaped correctly (to U+0127: 'ħ')
    assert.equal(unescapeOneQuadString('\\u0127'), '\u{0127}');
```